devops/pipeline/e2etest.yaml

```diff
@@ -345,6 +345,6 @@ stages:
 
       - script: |
           terraform init
-          terraform destroy -var client_id="$(CLIENT_ID)" -var client_secret="$(CLIENT_SECRET)"  -var subscription_id="$(SUBSCRIPTION_ID)" -var tenant_id="$(TENANT_ID)" -var key_vault_id="$(KEY_VAULT_ID)" -auto-approve | true
+          terraform destroy -var client_id="$(CLIENT_ID)" -var client_secret="$(CLIENT_SECRET)"  -var subscription_id="$(SUBSCRIPTION_ID)" -var tenant_id="$(TENANT_ID)" -var key_vault_id="$(KEY_VAULT_ID)" -auto-approve || true
```

Comment:
Has this change been tested in a private branch, to fix issues with resource cleanup?

Reply:
i have not, but the setup requires a few changes to the pipeline which i rather not do just to test this change. I have high confidence this will work.